src/openforms/forms/api/serializers/form_variable.py

```diff
@@ -110,6 +176,7 @@ class FormVariableSerializer(serializers.HyperlinkedModelSerializer):
     service_fetch_configuration = ServiceFetchConfigurationSerializer(
         required=False, allow_null=True
     )
+    prefill_options = FormVariableOptionsSerializer(required=False)
```

Comment:
at this level, this must remain a plain `JSONField`, but we should add validators that it must always be a dict (so list, int, string etc. JSON values are not allowed).

In the `def validate(...)` method we can/must then look up the specific validation options based on the value of the `plugin` field so that we can perform deep validation. You can take a look at `FormRegistrationBackendSerializer` for inspiration, it does the same from the registration plugin perspective